transcom/mymove
# INT B-18992 reweigh email shipment id

## Agility Ticket
[B-18992](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18992)

## Summary

There are three ways to trigger the reweigh email and all three have been verified.

1. Request reweigh as a TOO under 'Move task order' tab.
2. When shipment weight is over the max weight allowance.
3. When shipment weight is within 10% of the max weight allowance.

Easiest method to test this email update is to use 'Request reweigh' as a TOO.

- Updated the `reweigh_requested_template.html/txt ` to match the content and format of the email doc provided in Agility.
- Title of the email has been updated to include the Shipment # which is first 8 characters of the `ShipmentID`.


### How to test

1. Use this command to start your server with email enabled: 
`EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run`
2. Use "HHGMoveInSITEndsTomorrow" testharness to create a move.
3. Login as a Customer for that move and edit the email to your email.
4. Login as a TOO and find the move -> go to 'Move task order' tab -> Scroll down to find 'Request reweigh' and click.
5. Should receive an email to your added email.
6. While on the move page, use the developer tools to find the ShipmentLocator field, which should be in the "mto_shipments" request under the "Network" tab.
7. Make sure the subject line contains the Shipment Locator for the move instead of the old ID.
